File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/server/net/ServerPlayerPacketHandler.java
package pize.tests.voxelgame.server.net;

import pize.net.tcp.packet.PacketHandler;
import pize.tests.voxelgame.main.net.packet.*;

public interface ServerPlayerPacketHandler extends PacketHandler{
    
    void handleChunkRequest(SBPacketChunkRequest packet);
    
    void handlePlayerBlockSet(SBPacketPlayerBlockSet packet);
    
    void handleMove(SBPacketMove packet);
    
    void handleRenderDistance(SBPacketRenderDistance packet);
    
    void handleSneaking(SBPacketPlayerSneaking packet);
    
    void handleChatMessage(SBPacketChatMessage packet);
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/server/net/ServerPlayerPacketHandler.kt
package pize.tests.voxelgame.server.net

import pize.net.tcp.packet.PacketHandler
import pize.tests.voxelgame.main.net.packet.*

interface ServerPlayerPacketHandler : PacketHandler {
    fun handleChunkRequest(packet: SBPacketChunkRequest)
    fun handlePlayerBlockSet(packet: SBPacketPlayerBlockSet)
    fun handleMove(packet: SBPacketMove)
    fun handleRenderDistance(packet: SBPacketRenderDistance)
    fun handleSneaking(packet: SBPacketPlayerSneaking)
    fun handleChatMessage(packet: SBPacketChatMessage)
}
